video/out/opengl/context_wayland.c

```diff
@@ -56,7 +56,23 @@ static void resize(struct ra_ctx *ctx)
 static void wayland_egl_swap_buffers(struct ra_ctx *ctx)
 {
     struct priv *p = ctx->priv;
+    struct vo_wayland_state *wl = ctx->vo->wl;
     eglSwapBuffers(p->egl_display, p->egl_surface);
+    wayland_sync_swap(ctx->vo, wl->ust, wl->msc, wl->sbc);
+}
+
+static void wayland_egl_get_vsync(struct ra_ctx *ctx, struct vo_vsync_info *info)
+{
+    struct vo_wayland_state *wl = ctx->vo->wl;
+    if (wl->presentation) {
+        info->vsync_duration = wl->vsync_duration;
+        info->skipped_vsyncs = wl->last_skipped_vsyncs;
+        info->last_queue_display_time = wl->last_queue_display_time;
+    } else {
+        info->vsync_duration = -1;
+        info->skipped_vsyncs = -1;
+        info->last_queue_display_time = -1;
```

Comment:
You can just not do anything. info will be setup for "no information" by default.